.github/workflows/advertisements-fixed.yml

```diff
@@ -15,23 +17,43 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout Code
+    - name: Checkout
       uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v1
-    - name: Setup Docker Buildx
+
+    - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-    - name: Login to Docker Hub
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Login to DockerHub
       uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
-    - name: Build and Push Docker Image
+
+    - name: Login to ECR
+      id: login-ecr
+      uses: docker/login-action@v1
+      with:
+        registry: public.ecr.aws
+        username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    - name: Build and push
       uses: docker/build-push-action@v2
       with:
         context: ./ads-service-fixed
-        platforms: linux/amd64 # TODO: Support linux/arm64?
+        platforms: linux/amd64
         push: true
-        tags: ddtraining/advertisements-fixed:latest
+        tags: |
+          ddtraining/advertisements-fixed:latest
+          public.ecr.aws/b1o7r7e0/ddtraining/ads-service-fixed:latest
```

Comment:
why are we changing the name of the image based on the repository? `advertisements-fixed` in dockerhub and `ads-service-fixed` in ECR? we should keep the same name

Reply:
Yes, you're right. These are some changes I forgot to update. I worked with other ECR repo during the dev in order to keep the official repo clean.